### DISCUSSION TITLE
Let snakes be public outside of tournament matches

### DISCUSSION THREAD

**Discussion by BoldBigflank** [BattlesnakeOfficial/feedback#185] [Feature Requests]
In the Battlesnake edit screen you have the option to let your snake be public so others can recruit them to battle.

![image](https://user-images.githubusercontent.com/636964/154817109-d8579e30-e7b2-4a99-bc45-107c343253d3.png)

Now that the spring league is grouped to the morning hours, I think it would be nice to change this to a select dropdown so that we can choose "Allow", "Only outside of active tournament matches", and "None"

**Comment by bvanvugt** [comment]
This is really compelling, and we have a few ideas coming in the next few months that will help get us here :)
